pkg/operator/ceph/cluster/controller.go

```diff
@@ -780,6 +782,53 @@ func (c *ClusterController) onDeviceCMUpdate(oldObj, newObj interface{}) {
 // Delete event functions
 // ************************************************************************************************
 
+func (c *ClusterController) dataDirHostPathCleanUp(clust *cephv1.CephCluster) {
+	logger.Infof("starting data dir host path clean up for cluster - %+v", clust)
+	command := []string{"rm", "rf", clust.Spec.DataDirHostPath}
+	volumes := []v1.Volume{}
+	volumeMounts := []v1.VolumeMount{}
+	validNodes := k8sutil.GetValidNodes(clust.Spec.Storage, c.context.Clientset, cephv1.GetOSDPlacement(clust.Spec.Placement))
+	logger.Infof("valid nodes - %+v", validNodes)
+	for _, node := range validNodes {
+		logger.Infof("cleaning up hostpath for node  %q", node.Name)
+		myVolume := v1.Volume{Name: "test", VolumeSource: v1.VolumeSource{HostPath: &v1.HostPathVolumeSource{Path: clust.Spec.DataDirHostPath}}}
+		volumes = append(volumes, myVolume)
```

Comment:
This is going to append a volume and volume mount to each job for each host such that the first job will have one volume/mount, and the Nth job will have N volumes/mounts. The volumes/mounts should be the same for every node, and should be set before the `for` loop where currently there are just empty slices.